src/Symfony/Component/Cache/StatsInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache;
+
+/**
+ * Subset of the Doctrine\Common\Cache\Cache interface.
+ */
+interface StatsInterface
+{
+    const STATS_HITS = 'hits';
+    const STATS_MISSES = 'misses';
+    const STATS_UPTIME = 'uptime';
+    const STATS_MEMORY_USAGE = 'memory_usage';
+    const STATS_MEMORY_AVAILABLE = 'memory_available';
+
+    /**
+     * Retrieves cache statistics from the data store.
+     *
+     * The statistics array has the following values:
+     * - hits: Number of keys that have been requested and found present.
+     * - misses: Number of items that have been requested and not found.
+     * - uptime: Time that the server is running.
+     * - memory_usage: Memory used by the server to store items.
+     * - memory_available: Memory allowed to use for storage.
```

Comment:
The description for `memory_available` looks confusing to me. Is it the total memory allowed or the remaining memory available?